File: 백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.py
a = int(input())
b = int(input())

i = b - a

if i >= 1 and i <= 20:
    print("You are speeding and your fine is $100.")

elif i >= 21 and i <= 30:
    print("You are speeding and your fine is $270.")

elif i >= 31:
    print("You are speeding and your fine is $500.")

else:
    print("Congratulations, you are within the speed limit!")

File: 백준/Bronze/6763. Speed fines are not fine！/Speed fines are not fine！.c
#include <stdio.h>

int main(void) {
  int a, b;
  scanf("%d", &a);
  scanf("%d", &b);

  int i = b - a;

  if(i >= 1 && i <= 20){
    printf("You are speeding and your fine is $100.");
  }
  else if(i >= 21 && i <= 30){
    printf("You are speeding and your fine is $270.");
  }
  else if(i >= 31){
    printf("You are speeding and your fine is $500.");
  }
  else{
    printf("Congratulations, you are within the speed limit!");
  }
  
  return 0;
}